### PR TITLE
Update python

### PR DESCRIPTION
**Pull request by tianon**
Changes:

- https://github.com/docker-library/python/commit/811625e: Merge pull request https://github.com/docker-library/python/pull/954 from infosiftr/really-no-setuptools-or-wheel
- https://github.com/docker-library/python/commit/a8ec33a: Stop installing setuptools in python 3.12+
- https://github.com/docker-library/python/commit/5ee49c2: Update 3.9 to get-pip https://github.com/pypa/get-pip/commit/def4aec84b261b939137dd1c69eff0aabb4a7bf4
- https://github.com/docker-library/python/commit/0dabaf3: Update 3.8 to get-pip https://github.com/pypa/get-pip/commit/def4aec84b261b939137dd1c69eff0aabb4a7bf4
- https://github.com/docker-library/python/commit/14e03d8: Update 3.13-rc to get-pip https://github.com/pypa/get-pip/commit/def4aec84b261b939137dd1c69eff0aabb4a7bf4
- https://github.com/docker-library/python/commit/109e83a: Update 3.12 to get-pip https://github.com/pypa/get-pip/commit/def4aec84b261b939137dd1c69eff0aabb4a7bf4
- https://github.com/docker-library/python/commit/625a0a3: Update 3.11 to get-pip https://github.com/pypa/get-pip/commit/def4aec84b261b939137dd1c69eff0aabb4a7bf4
- https://github.com/docker-library/python/commit/e84c3f7: Update 3.10 to get-pip https://github.com/pypa/get-pip/commit/def4aec84b261b939137dd1c69eff0aabb4a7bf4

### CODE REVIEW

**Comment by tianon** [comment]
Reviewing this now is good, but we should wait until Tuesday to merge it so we don't break everyone over the weekend / US+CA holiday.

**Review by yosifkit — APPROVED** [review]

**Pull request merged**
